src/main/groovy/org/pih/warehouse/inventory/CycleCountUpdateItemCommand.groovy

```diff
@@ -23,7 +25,7 @@ class CycleCountUpdateItemCommand implements Validateable {
     Date dateCounted
 
     def beforeValidate() {
-        String cycleCountItemId = RequestContextHolder.getRequestAttributes().params?.cycleCountItemId
+        String cycleCountItemId = RequestContextHolder.getRequestAttributes().params?.cycleCountItemId ?: id
```

Comment:
just describing this for myself because I was confused initially.

So if the cc item id is in the URI, use that, else assume it's in the request payload (and if its in both, use the one in the URI). And I'm assuming we support both cases because we still have the non-batched API, which puts it in the URI.